1_fetch.R

```diff
@@ -255,6 +255,34 @@ p1_targets_list <- list(
     p1_depth_to_bedrock_tif,
     Shangguan_dtb_cm_250m_clip_path,
     format = "file"
+  ),
+  
+  # Dataset build in consultation with GW subject matter expert. table found in sharepoint in project data folder
+  # Dataset need to be downloaded from sharepoint location and placed into 1_fetch/in/ in order to build this target
+  tar_target(
+    p1_coarse_sediment_unitname_xwalk,
+    read_csv('1_fetch/in/surficial_materials_CONUS_unit_names.csv',
+             col_types = 'c'
+    )
+  ),
+  
+  # load in Soller et al. 2009's surficial material dataset and filter using xwalk table
```

Comment:
On second thought, the file `1_fetch/src/download_file.R` contains a function `download_file()` that is a wrapper for `utils::download.file()`. Would it work to download these shape files using the url from the web? e.g.:

```r
  tar_target(
    p1_soller_surficial_mat_zip,
    download_file("https://pubs.usgs.gov/ds/425/USGS_DS_425_SHAPES.zip",
                  fileout = "1_fetch/out/USGS_DS_425_SHAPES.zip", 
                  mode = "wb", quiet = TRUE),
    format = "file",
  ),
```

Reply:
My tendency has been to put anything that's generated using code in the `1_fetch/out` folder. In this case, I used `out` since it's an output of the `download_file()` function. With this in mind, I don't use `in` very much, because hopefully we can access most of the input data we need programmatically. Curious to hear your thoughts on that!